MobileCenter/MobileCenter/Internals/Utils/MSUtil.m

```diff
@@ -92,4 +94,25 @@ + (NSTimeInterval)nowInMilliseconds {
   return ([[NSDate date] timeIntervalSince1970] * 1000);
 }
 
+#pragma mark - Format Utility Methods
+
++ (NSString *)formatToUUIDString:(NSString *)aString {
+  NSMutableString *stringToFormat = [aString mutableCopy];
+  short dashesCount = (sizeof kMSUUIDDashIndexes) / (sizeof kMSUUIDDashIndexes[0]);
+
+  // Pre-validate string.
+  if (aString.length != (NSUInteger)(kMSUUIDLength - dashesCount)) {
+    return nil;
+  }
+  for (short i = 0; i < dashesCount; i++) {
```

Comment:
Minor thing but you can iterate from at the end of string to modify. This case the index would have better readability like `8, 12, 16, 20`.